doc/src/dump_image.rst

```diff
@@ -6,6 +6,9 @@ dump image command
 dump movie command
 ==================
 
+dump_modify command for image/movie options
+===========================================
```

Comment:
@sjplimp I don't think that this is a good place for this header since the "syntax" section below does **not** correspond to this command. That is why I added this later., where the "for image/movie options" test could be added for clarity and so that it will be different from the regular "dump_modify" entry in the navigation bar on the left.